crypto/random.go

```diff
@@ -29,7 +29,18 @@ func CRandHex(numDigits int) string {
 	return hex.EncodeToString(CRandBytes(numDigits / 2))
 }
 
-// Returns a crand.Reader.
+// CRandSeed returns a seed from the OS's randomness.
+func CRandSeed(length int) int64 {
+	b := randBytes(length)
+	var seed uint64
+	for i := 0; i < 8; i++ {
+		seed |= uint64(b[i])
+		seed <<= 8
+	}
+	return int64(seed)
+}
```

Comment:
One more thing: can we delete this? It's not used anywhere.

Reply:
Yes